module/workflow/src/Application/Controller/Api/Transition/TransitionDeleteAction.php

```diff
@@ -83,17 +83,16 @@ public function __construct(CommandBusInterface $commandBus)
      * @ParamConverter(class="Ergonode\Workflow\Domain\Entity\Status", name="source")
      * @ParamConverter(class="Ergonode\Workflow\Domain\Entity\Status", name="destination")
      *
-     * @param Workflow $workflow
-     * @param Status   $source
-     * @param Status   $destination
+     * @param AbstractWorkflow $workflow
+     * @param Status           $source
+     * @param Status           $destination
```

Comment:
I don't understand:
double `ParamConverter` to `Status` and no  `ParamConverter` to `AbstractWorkflow` 

I think about params `Status` is to delete